src/main/java/team/chisel/ctm/client/texture/IMetadataSectionCTM.java

```diff
@@ -145,17 +145,16 @@ public static IMetadataSectionCTM fromJson(JsonObject obj) throws JsonParseExcep
         }
     }
     
-    public static class Serializer implements IMetadataSectionSerializer<IMetadataSectionCTM> {
+    class Serializer implements MetadataSectionSerializer<IMetadataSectionCTM> {
 
         @Override
-        public @Nullable IMetadataSectionCTM deserialize(@Nullable JsonObject json) throws JsonParseException {
+        public @Nullable IMetadataSectionCTM fromJson(@Nullable JsonObject json) throws JsonParseException {
             if (json != null && json.isJsonObject()) {
                 JsonObject obj = json.getAsJsonObject();
                 if (obj.has("ctm_version")) {
                     JsonElement version = obj.get("ctm_version");
                     if (version.isJsonPrimitive() && version.getAsJsonPrimitive().isNumber()) {
-                        switch (version.getAsInt()) {
-                        case 1:
+                        if (version.getAsInt() == 1) {
```

Comment:
This was a switch to make diffs cleaner for future version additions

Reply:
okay, will undo that change